Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM nginx:1.11-alpine
+FROM nginx:1-alpine
```

Comment:
question: do you need to change the apline image version, and isn't it older now?

Reply:
`1-alpine` is newer than `1.11-alpine` because since only the major version is specified the newest `1.xx` version will be used. I change this since it's always good to have new nginx for security.